### PR TITLE
Turns the simple static html into a jade palace.

### PR DESCRIPTION
**Pull request by olizilla**
`npm run build` will build the site from the `src` dir, and squirt it out to the `dist` dir.
`npm run watch` will rebuild all the things as you edit them.
`npm run start` will launch a live-reload friendly server over `dist`.

**DO NOT MERGE TO MASTER UNTIL https://github.com/tableflip/tableflip-infrastructure IS UPDATED**

One for discussion. I'm not totally convinced this is better, but it does demo using the fancy new jade `include:transform` to convert markdown, or any other jstransformer on the fly.

Also `npm-run-all` is neato. It let's you chain parallel and sequential npm tasks to run.

### CODE REVIEW

**Comment by liamwooding** [comment]
@olizilla "DO NOT MERGE TO MASTER UNTIL https://github.com/tableflip/tableflip-infrastructure IS UPDATED" - did it get updated?

File: src/intimate-objects.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Intimate Objects
+
+TABLEFLIP were comissioned by Intimate Objects to build a website and CMS allowing visitors to design, create and share intimate objects with each other.
+
+<div class="full">
```
Comment (liamwooding) [inline src/intimate-objects.md:5]: What's the benefit of having these files be markdown instead of jade templates?

**Comment by olizilla** [comment]
No, the infrastructure project is not yet updated. It would need to know to build out the site. It currently assumes a static site.

File: src/intimate-objects.md (hunk 1 above)
Comment (olizilla) [inline src/intimate-objects.md:5]: In general, I'd prefer to keep "content" as markdown where ever possible. In this case the benefit is less clear I realised I needed to add the `.full` div to keep the expando magic. My assumption at the time was we'd figure out another solution for the expando at some point, so the div could be removed from the markdown.

**Pull request merged**